README.md

```diff
@@ -235,4 +235,16 @@ We can help each other as contributing and sharing.
 #### *Past*
 | Language | Date | Event Host | Event Name  | Event Link | 
 | --- | --- | --- | --- | --- |
-| :tr: | 31.03.2020 | IoT Türkiye | Kahve Sohbetleri (Online Etkinlik) | [link](https://ioturkiye.com/2020/03/kahve-sohbetleri-31-mart-2020-online-etkinlik/) |
\ No newline at end of file
+| :tr: | 31.03.2020 | IoT Türkiye | Kahve Sohbetleri (Online Etkinlik) | [link](https://ioturkiye.com/2020/03/kahve-sohbetleri-31-mart-2020-online-etkinlik/) |
+		
+
+## Azure
```

Comment:
Azure'u kategoriler kısmına da bir başlık olarak ekleyebilir misiniz? Dilerseniz ben de ekleyebilirim.
Teşekkürler destekleriniz için.